### PR TITLE
added Documentation

### PR DESCRIPTION
**Pull request by MurtazaBagwala07**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+            <div class="nav-action-btns">
+                <a class="a-link" href="index.html"><button class="nav-action-btn">
+                    Home
+                </button></a>
+                <a class="a-link" href="documentation/documentation.html"><button class="nav-action-btn">
+                    Documentation
+                </button></a>
+            </div>
+            <ul class="nav-social-container">
+                <a class="a-link" href="https://github.com/MurtazaBagwala07" target="_blank"><li class="nav-social-link"><i class="fab fa-github"></i></li></a>
```
Comment (ravikumar1002) [inline index.html:26]: after ul tag,  li will be next direct tag  not a tag

**Review by ravikumar1002 — COMMENTED** [review]
one suggestion, various places you used class name for single value, instead of making class name you can use utility class (like fs-md , d-none)

**Review by ravikumar1002 — APPROVED** [review]
class name are good 
well formatted document

looks good to merge

**Review by Logan1x — APPROVED** [review]
LGTM

**Pull request merged**